File: mobile/app/src/main/java/com/example/eduapp/etudiant/models/Department.java
package com.example.eduapp.etudiant.models;

import java.util.List;

public class Department {
    public int id;
    public String name;
    public String responsable;

    public List<Classe> classes;
    public List<Salle> salles;
    public List<Prof> profs;
}

File: Admin/Admin/Models/Department.cs
namespace Admin.Models
{
    public class Department
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Responsable { get; set; }

        public ICollection<Classe>? Classes { get; set; }
        public ICollection<Salle>? Salles { get; set; }
        public ICollection<Prof>? Profs { get; set; }

    }
}
